scripts/linters/pylint_extensions.py

```diff
@@ -1918,7 +1855,6 @@ def register(linter):
     linter.register_checker(SingleNewlineAboveArgsChecker(linter))
     linter.register_checker(DivisionOperatorChecker(linter))
     linter.register_checker(SingleLineCommentChecker(linter))
-    linter.register_checker(DocstringChecker(linter))
     linter.register_checker(BlankLineBelowFileOverviewChecker(linter))
     linter.register_checker(NewlineBelowClassDocstring(linter))
```

Comment:
Move this check as well to the DocstringParamterChecker class.

Reply:
Done.